src/__tests__/gatsby-node.tests.js

```diff
@@ -524,3 +525,267 @@ test('fetchPloneBreadcrumbs returns Gatsby Node for Breadcrumbs', async () => {
   );
   expect(data).toEqual(fetchBreadcrumbsNode);
 });
+const generatorid = 'http://localhost:8080/Plone/this-is-from-alok';
+const generatorbaseUrl = 'http://localhost:8080/Plone';
+let generatorexpansions;
+const generatorbacklinks = new Map();
+generatorbacklinks.set('/', ['']);
+generatorbacklinks.set('/examples/event-collection/', ['']);
+generatorbacklinks.set('/examples/the-news-collection/', ['']);
+generatorbacklinks.set('/examples/midsummer.jpg/', [
+  '/examples/the-news-collection/',
+]);
+generatorbacklinks.set('/this-is-from-alok/', ['']);
+
+const mockDataPloneNodeGenerator = {
+  '@components': {
+    actions: { '@id': 'http://localhost:8080/Plone/@actions' },
+    breadcrumbs: {
+      '@id': 'http://localhost:8080/Plone/@breadcrumbs',
+      items: [],
+    },
+    navigation: {
+      '@id': 'http://localhost:8080/Plone/@navigation',
+      items: [Array],
```

Comment:
What is `Array` here?

Reply:
I am getting mock data by logging what I am getting from the fetchPlone function present in ploneNodeGenerator. I think this array contains all the navigation of entire site like 
```Javascript
items: [
    {
      description: '',
      title: 'Home',
      _path: '/',
      _id: 'http://localhost:8080/Plone',
    },
    {
      description: '',
      title: 'gatsby-source-plone',
      _path: '/index/',
      _id: 'http://localhost:8080/Plone/index',
    },
    {
      description: 'News on gatsby-source-plone development',
      title: 'News',
      _path: '/news/',
      _id: 'http://localhost:8080/Plone/news',
    },
    {
      description:
        'A folder with different standard content types that Plone supports out of the box',
      title: 'Examples',
      _path: '/examples/',
      _id: 'http://localhost:8080/Plone/examples',
    },
    {
      description: '',
      title: 'Tutorial',
      _path: '/tutorial/',
      _id: 'http://localhost:8080/Plone/tutorial',
    },
    {
      description: 'Reference documentation',
      title: 'Docs',
      _path: '/reference/',
      _id: 'http://localhost:8080/Plone/reference',
    },
    {
      description: 'This is newsItem',
      title: 'This is newsItem',
      _path: '/this-is-newsitem/',
      _id: 'http://localhost:8080/Plone/this-is-newsitem',
    },
    {
      description: 'Lorum Ipsum I am figuring out',
      title: 'This is newsItem2',
      _path: '/this-is-newsitem2/',
      _id: 'http://localhost:8080/Plone/this-is-newsitem2',
    },
  ],
  _path: '/',
  _id: 'http://localhost:8080/Plone/@navigation',
};
```

Reply:
After a lot of thinking and investing a large amount of time around(1day) I found that since I am doing console.log it is not logging in-depth array or object logging because Json.stringify shows a different result. I think that I got the solution but Implementation is still left.

Reply:
Finally test pass :)